neural_style.py

```diff
@@ -200,7 +201,19 @@ def imread(path):
 
 def imsave(path, img):
     img = np.clip(img, 0, 255).astype(np.uint8)
-    Image.fromarray(img).save(path, quality=95)
+    im_obj = Image.fromarray(img)
+    try:
+        im_obj.save(path, quality=95)
+    except KeyError:
```

Comment:
Oh you're right, I didn't read your code closely enough. But yeah, I think whitelist is nicer.